context.go

```diff
@@ -151,17 +151,21 @@ func (c *Context) setDefaultTags() error {
 	return nil
 }
 
-// Stats gets stats for a context as a string
-func (c *Context) Stats() (string, error) {
+// Stats gets stats for a context as json
```

Comment:
1. I would describe this as "JSON bytes" to be abundantly clear.
2. Why the switch from a string to bytes?

Reply:
1. Updated comment
2. Switching to bytes provides more compatibility with the normal encoding/json package as it is the common container for most actions, e.g things like json.RawMessage, decoding etc